src/Plate/index.tsx

```diff
@@ -210,16 +210,16 @@ function Well(props: { position: number; well?: PlateWell }) {
     <span
       style={{
         backgroundColor: props.well?.selected
-          ? MLL_THEME.warningColor
-          : MLL_THEME.tableBorderColor,
+          ? THEME.warningColor
+          : THEME.tableBorderColor,
```

Comment:
Redefine in terms of the palette? @SimBig please check other usages where the theme colors are used and the semantics don't line up.